### PR TITLE
[IS-1086] remove deprecated grpc interfaces

### PR DESCRIPTION
**Pull request by yakkle**
- remove deprecated services, rpcs in loopchain.proto
 - remove not using grpc-related codes

releated PR : https://github.com/icon-project/icon-rpc-server/pull/135

### CODE REVIEW

**Review by hyerim-kim — APPROVED** [review]

File: loopchain/peer/peer_outer_service.py [hunk 1]
```diff
@@ -3,10 +3,11 @@
 import asyncio
 import copy
 import json
-import logging
 import math
 import time
```
Comment (metalg0su) [inline loopchain/peer/peer_outer_service.py:8]: Need to optimize import lines.
Delete blank please.

**Review by metalg0su — APPROVED** [review]
Thank you for your precious job!
This PR should be merged ASAP.

**Comment by metalg0su** [comment]
@yakkle 

https://github.com/icon-project/icon-rpc-server/pull/135 looks related to this PR .
Should https://github.com/icon-project/icon-rpc-server/pull/135 be merged before this PR?

**Comment by yakkle** [comment]
> @yakkle
> 
> [icon-project/icon-rpc-server#135](https://github.com/icon-project/icon-rpc-server/pull/135) looks related to this PR .
> Should [icon-project/icon-rpc-server#135](https://github.com/icon-project/icon-rpc-server/pull/135) be merged before this PR?

It's not necessary to merge first.
It would be best to merge at the same time.

**Review by windies21 — APPROVED** [review]

**Pull request merged**